src/plot_api/plot_api.js

```diff
@@ -244,8 +244,6 @@ exports.plot = function(gd, data, layout, config) {
                     'position': 'absolute',
                     'top': 0,
                     'left': 0,
-                    'width': '100%',
-                    'height': '100%',
```

Comment:
I don't think those two lines were ever needed since a few lines later we set the size of the canvas via its attributes anyway:
https://github.com/plotly/plotly.js/blob/f8c959d3c2f9c7be683754a9422b1b3cda7e032e/src/plot_api/plot_api.js#L253-L255

Reply:
Nice find!